EasyPost.Tests/ParametersTests/ParametersTest.cs

```diff
@@ -71,13 +72,13 @@ public void TestParametersToDictionaryWithSubDictionary()
                 Street1 = streetA,
             };
 
-            EasyPost.BetaFeatures.Parameters.Addresses.Create newAddressCreationParameters = new EasyPost.BetaFeatures.Parameters.Addresses.Create
+            Create newAddressCreationParameters = new Create
```

Comment:
issue: this should be the full namespace here

Reply:
Oh, right, the unit tests...